packages/react/src/index.ts

```diff
@@ -16,7 +16,7 @@ export { default as Attachment, AttachmentProps } from './components/Attachment/
 
 export { default as Avatar, AvatarProps } from './components/Avatar/Avatar'
 
-export { default as Box, BoxProps } from './components/Box/Box'
+export { default as Box, BoxProps } from './components/Box/BoxHeavy'
```

Comment:
why do we export `BoxHeavy` and what does it bring in addition to the new `Box` that we still need it?

Reply:
the main reason is mentioned in the description:

```
To prevent breaking changes, this PR has preserved original `Box` component's functionality _exported_ from Stardust.
```

Essentially, we would like to prevent Stardust from breaking changes to the clients that might already used theme-based styling approach to `Box`. Yes, while `Box` theme-based styles are not introduced anywhere in Stardust, it might be the case that client had already introduced some style overrides that target `Box` component at places of custom client's usage.

Consider the following example:

```jsx
// this might be an approach taken to circumvent current Stardust problem
// of positioning support
<Box variables={{ absolutePositioned: true }}>
   {someContent}
</Box>
```

with client's styles
```js
export const boxStyles = {
  root: ({ variables }) => ({ 
     variables.absolutePositioned && {
        position: absolute
    }
  })
```

Note that in that case it is always the case that client _won't like to target inner Stardust `Box` elements_ - the intent is to introduce styles to only those boxes that are used on client side. If we would export light version of `Box` that doesn't support theme-based styling, then breaking change will be introduced to the client. However, if we would export the same Box component as before, there won't be any breaking changes happen